lib/private/twilio/send-message.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+
+
+  friendlyName: 'Send message',
+
+
+  description: 'Send a message using the Twilio SMS API',
+
+
+  moreInfoUrl: 'https://www.twilio.com/docs/sms',
+
+
+  inputs: {
+
+    accountSid: {
+      example: 'DZe5eafd3c69b1e74example5852c04a9102',
+      description: 'The "account sid" associated with your Twilio account.',
+      whereToGet: {
+        url: 'https://www.twilio.com/console',
+      },
+      required: true
+    },
+
+    authToken: {
+      example: 'Dafd4example5852c3c69bZe5e1e704a9102',
+      description: 'The "auth token" associated with your Twilio account.',
+      whereToGet: {
+        url: 'https://www.twilio.com/console',
+      },
+      required: true
+    },
+
+    from: {
```

Comment:
Does `from` and `to` match how it works in sendgrid?

Reply:
Input names match, but in sendgrid "to" is listed first; we talked about listing the things that can be globally configured first for this one, though, so I'm leaving this in the existing order at the moment. Let me know your thoughts when you review, I'll leave this one unresolved for now.